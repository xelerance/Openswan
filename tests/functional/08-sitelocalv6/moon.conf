# /etc/ipsec.conf - strongSwan IPsec configuration file

version 2

conn host-host
	left=fec0::1
	#leftcert=moonCert.pem
	leftid=@moon.strongswan.org
	right=fec0::2
	rightid=@sun.strongswan.org
	auto=add

