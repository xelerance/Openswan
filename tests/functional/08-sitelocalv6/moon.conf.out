opening file: moon.conf
#conn host-host loaded

version 2.0

config setup


# begin conn host-host
conn host-host
	left=fec0::1
	leftid="@moon.strongswan.org"
	right=fec0::2
	rightid="@sun.strongswan.org"
	auto=add
	type=tunnel
	compress=no
	pfs=yes
	rekey=yes
	overlapip=no
	authby=rsasig
	phase2=esp
# end conn host-host

# end of config
